VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/private-consultation-room-controls/private-consultation-room-controls.component.scss

```diff
@@ -111,6 +111,14 @@
   }
 }
 
+.small-button.green {
+  color: govuk-colour('white');
+  background: govuk-colour('green');
+  &:hover {
+    background: govuk-colour('green');
+  }
```

Comment:
Is this to override a hover style from another class?

Reply:
I have removed as it was not necessary